brefv/payloads/foxglove/Imu.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package foxglove;
+
+import "foxglove/Quaternion.proto";
+import "foxglove/Vector3.proto";
+import "foxglove/Header.proto";
+
+message Imu {
```

Comment:
I suggest to rename as `ImuReading` even tough it is named simply `Imu` in ROS. I think it better express the actual content.

Reply:
Fixed